app/assets/packs/application.js

```diff
@@ -0,0 +1,12 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
```

Comment:
Is there a way we can not include the fully compiled packages? We'll probably need to figure out a way to make it work on heroku, but committing compiled assets makes all change sets super noisy.

Reply:
This is webpacker's comments being confusing -- the COMPILED assets go in `/public/packs`, which is ignored. These files are what webpack calls "entry points" -- when it compiles assets, it uses entry points as the roots of dependency trees that it then compiles into production asset files. So this file is compiled by webpack in the sense that a Java compiler compiles `.java` files, not in the sense that a Java compiler compiles Java bytecode.

Webpacker should work out of the box on heroku as it adds a compilation rake task... I think. That's what it advertises itself as doing!